### PR TITLE
Add optional default scripts log suppression to the script debug window

### PR DESCRIPTION
**Pull request by huffman**

### CODE REVIEW

**Comment by tapalisa** [comment]
https://highfidelity.manuscript.com/f/cases/19511/Please-fix-logspam-in-Script-Log

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/14384/HighFidelity-Beta-PR14274-5c5a5c9.apk

File: scripts/developer/debugging/debugWindow.js [hunk 1]
```diff
@@ -19,6 +19,23 @@ if (scripts.length >= 2) {
     return;
 }
 
+var SUPPRESS_DEFAULT_SCRIPTS_MENU_NAME  = "Developer"
+var SUPPRESS_DEFAULT_SCRIPTS_ITEM_NAME = "Suppress messages from default scripts in Debug Window";
```
Comment (thoys) [inline scripts/developer/debugging/debugWindow.js]: 🏆 longest menu item award

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/14384/HighFidelity-Beta-PR14274-5c5a5c9.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/14384/HighFidelity-Beta-Interface-PR14274-5c5a5c9.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/14384/HighFidelity-Beta-PR14274-5c5a5c9.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/14384/HighFidelity-Beta-Interface-PR14274-5c5a5c9.exe

**Comment by QliCknox** [comment]
Currently Testing

**Comment by QliCknox** [comment]
Pass @conklin94122 

Logs seen initially, stop showing up when suppressing messages from default scripts in the debug window

**Pull request merged**